sparse/boot/flash.sh

```diff
@@ -194,7 +198,7 @@ if [ -z ${BLOB_BIN_PATH} ]; then
 fi
 
 BLOBS=""
-for b in $(ls -1 ${BLOB_BIN_PATH}/*_{v16,v17}_nile.img 2>/dev/null); do
+for b in $(ls -1 ${BLOB_BIN_PATH}*_{v16,v17}_nile.img 2>/dev/null); do
```

Comment:
This won't work if path is overridden without trailing slash, e.g. `export BLOB_BIN_PATH=~ ./flash.sh`

Reply:
Ah, will fix to different place

Reply:
Why to different? `${BLOB_BIN_PATH}*_{v16,v17}_nile.img` now looks like a filename :)

Reply:
Well if you flash normally, it will print .//SW... to blob name.